includes/admin/post-types/meta-boxes/class.llms.meta.box.award.certificate.sync.php

```diff
@@ -105,7 +105,8 @@ private function sync_action() {
 			'This action will replace the current title, content, and the background image with the template ones.\nAre you sure you want to proceed?',
 			'lifterlms'
 		);
-		$on_click   = "return confirm('${sync_alert}')";
+		$sync_alert = str_replace( "'", "\'", $sync_alert);
```

Comment:
Using [esc_js()](https://developer.wordpress.org/reference/functions/esc_js/) I believe is more thorough for escaping inline JS

Reply:
`esc_js()` is too thorough. It changes `\n` to `n`.

Reply:
I tried adding various numbers of backslashes to `\n` and then running the string through `esc_js()`. It wants `\n` to be output as `\\n`, but in the Brave web browser, `\\n` shows up as a literal '\n` in the pop-up dialog.

Reply:
`esc_attr__()` may be a better solution to escaping quotes.